dataset/src/main/scala/frameless/functions/WindowFunctions.scala

```diff
@@ -0,0 +1,26 @@
+package frameless.functions
+
+import frameless.{ OrderedWindow, TypedAggregate, TypedColumn, TypedWindow }
+import org.apache.spark.sql.{ functions => untyped }
+
+trait WindowFunctions {
+  import WindowFunctionsHelpers.dense_rankObj
+
+  def dense_rank() = dense_rankObj
```

Comment:
do we want to keep the same syntax as Spark where you call `dense_rank().over(windowSpec)` or just take the window spec as an argument to `dense_rank`? There's no valid `dense_rank` without a window spec.

Reply:
I don't have a strong opinion on this. Staying close to vanilla is a plus but here it really makes little sense :smile:. To be consistent I would CamelCase it either-way.